examples/stripe-script/toolpad/resources/functions.ts

```diff
@@ -0,0 +1,52 @@
+import fs from 'fs';
+import { exec } from 'child_process';
+import path from 'path';
+import Stripe from 'stripe';
+
+const initStripe = () => {
+  if (!process.env.STRIPE_TOKEN) {
+    throw new Error('Missing STRIPE_TOKEN environment variable');
+  }
+  return new Stripe(process.env.STRIPE_TOKEN, {
+    apiVersion: '2022-11-15',
+  });
+};
+
+// const stripe = new Stripe(process.env.STRIPE_TOKEN, {
+//   apiVersion: '2022-11-15',
+// });
+
+export async function listInvoices(limit: number = 100, starting_after?: string) {
+  const stripe = initStripe();
+  const list = await stripe.invoices.list({
+    limit,
+    starting_after,
+  });
+
+  return JSON.stringify(list.data);
```

Comment:
It's strange that we need to stringify this, it should just work.
Maybe the data returned by the function as an object is too big? Could we maybe try a lower limit or filter out some properties?
I don't have a Stripe token so just speculating, let me know if you want me to try it.

Reply:
It worked with just returning the `list` instead of `list.data` - not quite sure why but leaving this here to offer clarity in case of attempts of future resolution